freshmaker/pyxis_gql.py

```diff
@@ -47,10 +48,9 @@ def connect(self):
 class PyxisGQL:
     def __init__(self, url, cert):
         """Create authenticated Pyxis GraphQL session"""
+        pyxis_krb_auth = HTTPKerberosAuth(mutual_authentication=OPTIONAL, force_preemptive=True)
         transport = RequestsHTTPTransportWithCert(
-            url=url,
-            retries=3,
-            cert=cert,
+            url=url, retries=3, cert=cert, auth=pyxis_krb_auth, verify=False
```

Comment:
I don't remember why we have `verify=False`, we can remove this.

Reply:
I believe this flag turns off the SSL verification during the requests. If I don't set it to false, I run into the following error when I try to make any of the queries:

> `requests.exceptions.SSLError: HTTPSConnectionPool(host='graphql.pyxis.dev.engineering.redhat.com', port=443): Max retries exceeded with url: /graphql/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)'))) `

I believe this flag will be necessary.